microsoft-edge/devtools-guide-chromium/css/grid.md

```diff
@@ -148,9 +144,10 @@ The **Grid overlays** section contains a list of grids that are present on the p
 
 ### Enable overlay views of multiple grids
 
-To display the overlay grid for multiple grids, select the checkbox next to each name of the grid.  In the example, there are 2 grid overlays enabled, that are each represented with different colors:
+To display the overlay grid for multiple grids, select the checkbox next to each name of the grid.  In the example, there are 3 grid overlays enabled, that are each represented with different colors:
 
 *  `main`
```

Comment:
the picture doesn't show a checkbox labelled "main"; it's labelled "body" - reconcile?
@captainbrosset 
rendered section:
https://review.learn.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/css/grid?branch=pr-en-us-3248#enable-overlay-views-of-multiple-grids

Reply:
old dark-mode png used to show "main"; picture was updated but text needs update, it seems

Reply:
my comments & commits will move pngs up to be below 1st lead-in at top of section - this should help keep text in sync w/ updated pngs @captainbrosset